src/os/data/configdescriptor.js

```diff
@@ -290,7 +290,8 @@ os.data.ConfigDescriptor.prototype.reimport = function() {
  * @inheritDoc
  */
 os.data.ConfigDescriptor.prototype.getNodeUI = function() {
-  return /** @type {string} */ (this.getFirst('nodeUi')) || '<defaultlayernodeui></defaultlayernodeui>';
+  return /** @type {string} */ (this.getFirst('nodeUi')) ||
+      os.data.ConfigDescriptor.base(this, 'getNodeUI') + this.nodeUI;
```

Comment:
The `+ this.nodeUI` portion is incorrect. That's what will (most likely) be returned by the super call.